src/components/complaints/NewComplaintForm.js

```diff
@@ -0,0 +1,94 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import TextField from 'material-ui/TextField';
+
+export default class NewComplaintForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      report: '',
+      ocurrenceDate: '',
+      categoryId: '',
+      address: '',
+      latitude: '',
+      longitude: '',
+    };
+  }
+
+  onPressSaveButton() {
+    this.props.salvarDenuncia(this.state);
+  }
+
+  setProperty(event, property) {
+    const currentState = this.state;
+    this.setState({
+      ...currentState,
+      [property]: event.target.value,
+    });
+  }
+
+  render() {
+    return (
+      <div id="form-nova-denuncia">
+        <h1>Nova Denúncia</h1>
+        <label htmlFor="report">Detalhamento:</label>
+
+        <TextField
+          id="report"
+          value={this.state.report}
+          onChange={e => this.setProperty(e, 'report')}
```

Comment:
O legal de colocar o handleInputChange é que não precisaria ficar declarando uma função o tempo todo. Apenas colocar

`onChange={this.handleChange}`